data/mods/Dark-Skies-Above/items/weapons.json

```diff
@@ -0,0 +1,101 @@
+[
+  {
+    "id": "dkz_flamesword_salvaged",
+    "type": "TOOL",
+    "category": "weapons",
+    "looks_like": "zweihander",
+    "name": { "str": "salvaged xenoid great sword" },
+    "description": "A well built and beautifully decorated great-sword forged from a dense, unknown chrome metal.  Its ability to disgorge flame does not seem to respond to your will, but the cutting edge is still exceptionally sharp.",
+    "weight": "5400 g",
+    "volume": "3750 ml",
+    "longest_side": "129 cm",
+    "price": 310000,
+    "price_postapoc": 7500,
+    "bashing": 17,
+    "cutting": 40,
+    "material": [ "dkz_blend_mat" ],
+    "symbol": "/",
+    "color": "light_gray",
+    "charges_per_use": 1,
+    "max_charges": 50,
```

Comment:
What charges does this use? It seems to not have a storage slot nor connect to UPS as far as I can tell.

Reply:
removed